TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -439,6 +441,11 @@ public void DidChangeConfigurationParams(IEnumerable<string> arguments)
 
             //Dispose previous watcher before setting new ones
             _DepWatcher.Dispose();
+            _CopyWatcher.Dispose();
```

Comment:
It is unusual to Dispose object before using it. Maybe the method should be renamed Reset and separated from the .NET Dispose pattern ?